packages/datetime/src/helpers/mountWithApplication.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Application } from 'react-rainbow-components';
+
+export default function mountWithApplication(el, options) {
+    return mount(<Application>{el}</Application>, options).find(el.type);
+}
```

Comment:
it more readable when use `element` instead of el, in react-rainbow we are updating and adding eslint rules and now it will not let you add variable names with less than 3 characters